frontend/app/src/people/widgetViews/parts/StatusPill.tsx

```diff
@@ -74,9 +74,7 @@ export default function GithubStatusPill(props: GithubStatusPillProps) {
 
   return (
     <div style={{ display: 'flex', ...style }}>
-      <Pill isOpen={isOpen}>
-        <div>{isOpen ? 'Open' : 'Closed'}</div>
-      </Pill>
+      <Pill isOpen={isOpen}>{/* <div>{isOpen ? 'Open' : 'Closed'}</div> */}</Pill>
```

Comment:
can we just delete the pill itself, this seems like its in the github status pill and doing 
`grep -nr "GithubStatusPill ./src` I I see this

```
grep -nr "GithubStatusPill" ./src
./src/people/widgetViews/wantedViews/DesktopView.tsx:6:import GithubStatusPill from '../parts/StatusPill';
./src/people/widgetViews/wantedViews/DesktopView.tsx:89:              <GithubStatusPill
./src/people/widgetViews/wantedViews/MobileView.tsx:5:import GithubStatusPill from '../parts/StatusPill';
./src/people/widgetViews/wantedViews/MobileView.tsx:106:              <GithubStatusPill
./src/people/widgetViews/summaries/wantedSummaries/CodingMobile.tsx:5:import GithubStatusPill from '../../parts/StatusPill';
./src/people/widgetViews/summaries/wantedSummaries/CodingMobile.tsx:54:          <GithubStatusPill status={status} assignee={assignee} />
./src/people/widgetViews/summaries/wantedSummaries/CodingDesktop.tsx:6:import GithubStatusPill from '../../parts/StatusPill';
./src/people/widgetViews/summaries/wantedSummaries/CodingDesktop.tsx:63:              <GithubStatusPill status={status} assignee={assignee} style={{ marginRight: 25 }} />
./src/people/widgetViews/parts/StatusPill.tsx:1:import { GithubStatusPillProps } from 'people/interfaces';
./src/people/widgetViews/parts/StatusPill.tsx:58:export default function GithubStatusPill(props: GithubStatusPillProps) {
./src/people/interfaces.ts:215:export interface GithubStatusPillProps {
```
We might want to change the name of this component to something else like `AssigneeStatusPill`